sdks/python/setup.py

```diff
@@ -440,7 +440,7 @@ def get_portability_package_data():
               'google-cloud-datastore>=2.0.0,<3',
               'google-cloud-pubsub>=2.1.0,<3',
               'google-cloud-pubsublite>=1.2.0,<2',
-              'google-cloud-storage>=2.16.0,<3',
+              'google-cloud-storage>=2.18.2,<3',
```

Comment:
Note for others who come across this - this got bumped in a previous pr for our containers - https://github.com/apache/beam/blob/01100a3b2fe9af9f30192f31cefe4e8d180dc782/sdks/python/container/py310/base_image_requirements.txt#L71

Reply:
fantastic.